cloud.foundry.cli/src/main/java/cloud/foundry/cli/logic/diff/change/parsing/ValueChangeParsingStrategy.java

```diff
@@ -61,12 +68,19 @@ public ValueChangeParsingStrategy(ChangeParser changeParser) {
                     .collect(Collectors.toList());
         }
 
-        return Arrays.asList(new CfObjectValueChanged(change.getAffectedObject().get(),
+        log.verbose("Parsing change type", change.getClass().getSimpleName(), "to custom change type",
+                CfObjectValueChanged.class.getSimpleName(), "with property", valueChange.getPropertyName(), "and changed value from",
+                valueChange.getLeft(), "to", valueChange.getRight());
+        List<CfChange> cfChanges = Collections.singletonList(new CfObjectValueChanged(change.getAffectedObject().get(),
                 valueChange.getPropertyName(),
                 this.extractPathFrom(change),
                 Objects.toString(valueChange.getLeft(), null),
                 Objects.toString(valueChange.getRight(), null)
         ));
+        log.verbose("Parsing change type", change.getClass().getSimpleName(), "to custom change type",
```

Comment:
I would recommend to use debug logging here to prevent long and confusing output